easybuild/easyconfigs/p/PTESFinder/PTESFinder-1-intel-2017b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'Tarball'
+
+name = 'PTESFinder'
+version = '1'
+
+homepage = 'ihttps://sourceforge.net/projects/ptesfinder-v1/'
```

Comment:
@boegel - what's going on with this URL?

Reply:
Ah, typo, I'll drop the leading `i`, that's what you're referring to, right?